test/sql/index/art/vacuum/test_art_vacuum_integers.test_slow

```diff
@@ -5,8 +5,6 @@
 statement ok
 PRAGMA enable_verification
 
-load __TEST_DIR__/test_art_vacuum_integers.db
```

Comment:
Is this moved to a non-storage test on purpose?

Reply:
Or, I can add these two pragmas to the test instead and then add some manual checkpoints/restarts if we want to keep testing that in these tests. What do you think? 😄 
```sql
statement ok
PRAGMA disable_checkpoint_on_shutdown;

statement ok
PRAGMA wal_autocheckpoint='1TB';
```

Reply:
Fair enough :)